README.md

```diff
@@ -1,182 +1,163 @@
-
-It turns out, we're adding (with the `+` operator) things of two different _types_. `2` is a number; `"2"` is a string. So when we add them together, JavaScript recognizes that we can't do simple arithmetic — instead, it tries to make the two things compatible. In this case, it turns the first `2` into `"2"` and then _concatenates_ (pushes together) the two things — so we get `"22"`.
+### Strings
+Strings are how we represent text in JavaScript. A string consists of a matching pair of `'single quotes'`, `"double quotes"`, or `` `backticks` `` with zero or more characters in between:
+```js
+typeof "I am a string."
+//=> "string"
 
-One last example — before you press "enter" for this one, we want to think about what's going to happen. What will the result be?
+typeof 'Me too!'
+//=> "string"
 
-``` javascript
-"2" + 2
+typeof `Me three!`
+//=> "string"
 ```
 
-Have you thought about it? What did we learn above? Are you ready? Okay, press "enter".
-
-That's right, the result of `"2" + 2` is also `"22"`.
-
-Why all this mumbo jumbo with types? Well, let's think about it. As humans when we see `2345`, we provide context: we might think of this sequence of integers in our heads as "two thousand three hundred forty-five"; we might note immediately that the integers are consecutive. We understand that `2345 + 2` should give us `2347`.
-
-JavaScript can't do that. It sees `2345` and only knows that it's a number. Similarly, it sees _anything_ in quotation marks as a string. We provide the context for JavaScript according to the rules that JavaScript follows — one of those rules happens to describe what happens when we add a number and a string.
-
-## You're just not my type
-
-How do we know what types we're dealing with? JavaScript gives us the handy dandy `typeof`. We use it like so
-
-``` javascript
-typeof 2 // "number"
-typeof "2" // "string"
-typeof '2' // "string" — strings can be enclosed in single (') or double (") quotes
+Even empty strings are strings:
+```js
+typeof ""
+//=> "string"
 ```
 
-Pretty simple, right? You can think of `typeof` as a shortcut for telling you what kind of hole to look for in our imaginary, new-fangled shape-sorter.
-
-In addition to numbers and strings, JavaScript has the following _primitive_ types:
-
-- Boolean
-- Undefined
-- Null
-
-Enter the following commands in your console to get a feel for the different types:
+### Booleans
+As in Ruby, booleans can only be the logical values `true` or `false`. Booleans play a big role in control flows and looping in JavaScript.
+```js
+typeof true
+//=> "boolean"
 
-``` javascript
-typeof 1
-typeof 10
-typeof 1.123
+typeof false
+//=> "boolean"
 ```
 
-``` javascript
-typeof "Albert"
-typeof '123'
-typeof "What's my type?"
-```
+### Symbols
+Symbols are a relatively new data type (introduced in ES2015) that's primarily used as a way to add anonymous properties to objects. Don't worry about symbols for now.
```

Comment:
@jmburges:

>Is there a JS history lesson? Would they have any idea what ES2015 is?

Yes, that will be in the Intro to JavaScript lesson.

>"anonymous properties to objects" that's a gibberish sentence to most folks

I understand, but that's what they're used for. I think it's important to introduce symbols b/c anyone who googles "JavaScript data types" or the like is going to see them and fall down a rabbit hole. However, I don't see the point in taking any more time than this to explain what they are b/c students won't have to use symbols at all in our JS course.

Is the "Don't worry about this for now" not enough of a signal to move on?